vulnerabilities/models.py

```diff
@@ -104,7 +104,7 @@ class VulnerabilityReference(models.Model):
         max_length=1024, help_text="URL to the vulnerability reference", blank=True
     )
     reference_id = models.CharField(
-        max_length=50,
+        max_length=180,
```

Comment:
Let's make it an even 200:
```suggestion
        max_length=200,
```